Jenkinsfile

```diff
@@ -236,8 +236,7 @@ exit $status
                                 if (env.BRANCH_NAME == 'master') {
                                     additionalTags.add('develop')
                                 }
-
-                                if (! version ==~ /.*-SNAPSHOT/) {
+                                if (! (version ==~ /.*-SNAPSHOT/)) {
```

Comment:
```suggestion
                                if (version !=~ /.*-SNAPSHOT/) {
```

Reply:
This reads better, I'll update it here and the azure check